pkg/jobs/registry.go

```diff
@@ -1259,3 +1259,9 @@ func (r *Registry) unregister(jobID int64) {
 		delete(r.mu.jobs, jobID)
 	}
 }
+
+// NudgeAdoptionQueueForTest is used by tests to tell the registry that there is
+// a job to be adopted.
+func (r *Registry) NudgeAdoptionQueueForTest() {
```

Comment:
huh, github swallowed my comment, but I meant to note here that we usually _prefix_ public methods used for tests only with `Testing`, e.g. `TestingNudgeAdoptionQueue` or something

Reply:
Done.